### DISCUSSION TITLE
Can't login due to Verification

### DISCUSSION THREAD

**Discussion by AlisherSrc** [vercel/community#1757] [Help]
### Summary

Hello, I'm a student from Kazakhstan, I tried to sign up via email but when I tried to login I was asked to provide my phone number.  No matter how many times I tried, I never received the verification code.

Could you please point out to the problem here?

### Additional information

email:
alisherskw@gmail.com

**Comment by psyferblock** [comment]
same problem from lebanon. the number was not accepted.
